msteams-platform/messaging-extensions/how-to/search-commands/respond-to-search.md

```diff
@@ -10,7 +10,7 @@ ms.localizationpriority: medium
 
 [!include[v4-to-v3-SDK-pointer](~/includes/v4-to-v3-pointer-me.md)]
 
-After the user submits the search command, your web service receives a `composeExtensions/query` invoke message that contains a `value` object with the search parameters. The invoke is triggered with the following conditions:
+After the user submits the search command, your web service receives a `composeExtension/query` invoke message that contains a `value` object with the search parameters. The invoke is triggered with the following conditions:
```

Comment:
Hi @andrewconnell, 

The name of the property in manifest schema is "composeExtensions".
Sharing the link for your reference:
https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#composeextensions
<img width="428" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/141609042/7954232e-b3e2-478b-8ce3-f85feaf15cc0">

Reply:
Correct, but the change in the PR is not referencing the app manifest. The article is talking about the type of the invoke message the bot framework receives & sends to the web service. That's `composeExtension/query`. 

Look at the same doc I submitted the PR for. The PR fixes the typo in the first paragraph, but if you scroll down to the first JSON code listing, you'll see the correct usage:

```json
{
  "type": "invoke",
  "name": "composeExtension/query",
  ..
}
```

Reply:
Okay. Thanks for the clarity.